### ISSUE TITLE
Create Login System (large)

### ISSUE THREAD

**Issue by emmafherman** [emmafherman/FreshSelect#1]

**Comment by SoftwareInventions** [comment]
The password process seems pretty cumbersome; was this desired by your users?  Normally people just choose their own passwords.  

Also, here and elsewhere we need some description of the implementation environment.   Is this a web app, standalone, what language, what UI library, etc.

**Comment by SoftwareInventions** [comment]
One more comment in general.  I'll await your estimates, but this set does seem a little light for 10 ideal days of effort...

**Comment by emmafherman** [comment]
I agree - yes, the admin suggested they provide info for users, but I suppose they can know this without needing their credentials. 👍
